glotaran/analysis/matrix_calculation.py

```diff
@@ -129,9 +132,8 @@ def _calculate_matrix(matrix_function, dataset_descriptor, axis, extra, index=No
     args = {
         'dataset_descriptor': dataset_descriptor,
         'axis': axis,
+        'extra': extra,
     }
-    for k, v in extra:
```

Comment:
What was the prior functionality you removed here?
It seems 'extra' was already part of the _calculate_matrix function args prior to this change.

Reply:
I had a different scheme in mind with `extra` originally, but I altered it. Prey they I don't alter it further... )